assets/css/style.css

```diff
@@ -0,0 +1,93 @@
+body {
+    background: #ECEEF1 !important;
+    color: #4C4C4C;
+}
+.header-section {
+    margin-top: 7px;
+}
+.top-section {
+    margin-bottom: 25px
+}
+.align-center{
+    text-align: center;
+}
+.inline{
+    display: inline;
+}
+.top-section-margin{
+    margin: 5% 5px 5% 5px;
```

Comment:
This could be written as margin: 5% 5px 
But please don't use % for padding or margin.